src/prefect/cli/work_pool.py

```diff
@@ -299,6 +301,42 @@ async def clear_concurrency_limit(
         exit_with_success(f"Cleared concurrency limit for work pool {name!r}")
 
 
+@work_pool_app.command()
+async def get_default_base_job_template(
+    type: str = typer.Option(
+        None,
+        "-t",
+        "--type",
+        help="The type of work pool for which to get the default base job template.",
+    ),
+    file: str = typer.Option(
+        None, "-f", "--file", help="If set, write the output to a file."
+    ),
+):
+    """
+    Get the default base job template for a given work pool type.
```

Comment:
Do the docs in api-ref get autogenerated, or do we need to update manually?

Reply:
The `work-pool` command isn't in the API ref because it hasn't been added to `mkdocs.yaml`. The docs generated for these CLI functions are pretty useless, but this doc string will be used to generate the help text for the CLI.